packages/terra-menu/src/MenuItem.jsx

```diff
@@ -150,6 +171,7 @@ class MenuItem extends React.Component {
     } else {
       attributes.onClick = this.wrapOnClick;
       attributes.onKeyDown = this.wrapOnKeyDown(attributes.onKeyDown);
+      attributes.onKeyUp = this.onKeyUp();
```

Comment:
See comment above, but given the update I'd expect custom prop `onKeyUp` attributes to be pass like
```
 attributes.onKeyUp = this.wrapOnKeyUp(attributes.onKeyUp);
```